uint8 DEFAULT=0
uint8 USER_CONFIGURED=1
uint8 type
float64 p_gain
float64 i_gain
float64 d_gain